frontend/components/Domain/Recipe/RecipeRating.vue

```diff
@@ -83,7 +82,7 @@ export default defineNuxtComponent({
     });
 
     function updateRating(val?: number) {
-      if (!isOwnGroup.value) {
+      if (!isOwnGroup.value || typeof val !== "number" || val === 0) {
```

Comment:
Since this is either a number or undefined, could we simplify this?
```suggestion
      if (!isOwnGroup.value || !val) {
```